app/helpers/session_helper.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Session helper methods
+module SessionHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  # Remembers a user in a persistent session.
+  def remember(user)
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
```

Comment:
At the end, set the current user to the user sent in parameter.

Reply:
## User setter added